frontend/src/libs/components/router/router.tsx

```diff
@@ -0,0 +1,35 @@
+import { App, RouterProvider } from '~/libs/components/components.js';
```

Comment:
That's circular dependency. Router imported from components/components.js and has imports from the same file. 
Be careful with that, it can lead to hard to debug bugs.
Use direct import from `../app/app.js`, same for router provider

Reply:
thank you! update imports:
import { App } from '../app/app.js';
import { RouterProvider } from '../router-provider/router-provider.js';